e2e/tests/functional/forms.e2e.spec.js

```diff
@@ -91,6 +93,45 @@ test.describe('Persistence operations @addInit', () => {
     });
 });
 
+test.describe('Persistence operations @couchdb', () => {
+    test.use({ failOnConsoleError: false });
+    test('Editing object properties should generate a single persistence operation', async ({ page }) => {
+        test.info().annotations.push({
+            type: 'issue',
+            description: 'https://github.com/nasa/openmct/issues/5616'
+        });
+
+        await page.goto('./', { waitUntil: 'networkidle' });
+
+        // Create a new 'Clock' object with default settings
+        const clock = await createDomainObjectWithDefaults(page, {
+            name: `clock-${uuid()}`,
+            type: 'Clock'
+        });
+
+        // Count all persistence operations (PUT requests) for this specific object
+        let putRequestCount = 0;
+        page.on('request', req => {
+            if (req.method() === 'PUT' && req.url().endsWith(clock.uuid)) {
+                putRequestCount += 1;
+            }
+        });
+
+        // Open the edit form for the clock object
+        await page.click('button[title="More options"]');
+        await page.click('li[title="Edit properties of this object."]');
+
+        // Modify the display format from default 12hr -> 24hr and click 'Save'
+        await page.locator('select[name="mctControl"]').nth(1).selectOption({ value: 'clock24' });
```

Comment:
we should consider updating this locator

Reply:
I think we can improve this pretty easily, let me see what I can whip up. In general though, we do need to improve the selectors on our forms across the board.